jaxrs/src/main/java/com/cerner/beadledom/jaxrs/exceptionmapper/FailureExceptionMapper.java

```diff
@@ -0,0 +1,68 @@
+package com.cerner.beadledom.jaxrs.exceptionmapper;
```

Comment:
Sorry to ask for another change to this package but I actually think `exceptionmapping` would be the best package name.

Reply:
No worries! Changed the package name from `exceptionmapper` to `exceptionmapping`: https://github.com/cerner/beadledom/pull/45/commits/5f20764db12495ac1e321368a52939c7ce24444d